### PR TITLE
CNFT1-841_Coordinate_install_of_Geocoding_repo_with_Foundations_Team_&_get_a_single_API_query_running_through_Postman

### PR DESCRIPTION
**Pull request by JimBarrows-Enquizit**

### CODE REVIEW

**Comment by JimBarrows-Enquizit** [comment]
this is the bare minimum the ticket requires.

File: apps/geocoding-api/src/main/postman/CNFT1-841.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+		"_exporter_id": "26611326"
+	},
+	"item": [
+		{
+			"name": "Direct call to smarty",
+			"request": {
+				"method": "GET",
+				"header": [],
+				"url": {
+					"raw": "https://us-street.api.smartystreets.com/street-address?auth-id=02f5b868-b819-0333-6da9-b2220a435a2a&auth-token=nN6E3P3Qdj1b0WZGJhu8&license=us-core-cloud&street=1600+amphitheatre+pkwy&city=mountain+view&state=CA&candidates=10",
```
Comment (mpeels) [inline apps/geocoding-api/src/main/postman/CNFT1-841.postman_collection.json:15]: auth id and token

File: apps/geocoding-api/build.gradle [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        attributes 'API-Version': archiveVersion
+    }
+}
+
+jar {
+    //  packages the application and spring into one jar
+    enabled = false
+}
+
+dependencies {
```
Comment (mpeels) [inline apps/geocoding-api/build.gradle:22]: not sure what all this service will touch, but might be able to remove several of these dependencies

File: apps/geocoding-api/src/main/postman/CNFT1-841.postman_collection.json (hunk 1 above)
Comment (JimBarrows-Enquizit) [reply apps/geocoding-api/src/main/postman/CNFT1-841.postman_collection.json]: Whoops.  Removed both place.

File: apps/geocoding-api/build.gradle (hunk 2 above)
Comment (JimBarrows-Enquizit) [reply apps/geocoding-api/build.gradle:22]: Removed some

**Review by mpeels — APPROVED** [review]

**Pull request merged**